pkg/util/mount/mount_linux.go

```diff
@@ -323,17 +323,21 @@ func (mounter *SafeFormatAndMount) formatAndMount(source string, target string,
 
 	// Try to mount the disk
 	glog.V(4).Infof("Attempting to mount disk: %s %s %s", fstype, source, target)
-	err = mounter.Interface.Mount(source, target, fstype, options)
-	if err != nil {
+	mountErr := mounter.Interface.Mount(source, target, fstype, options)
```

Comment:
Sometimes I need to return `err` and sometimes `mountErr`, depending on what failed after mount check. IMO splitting it into smaller function would not solve this problem.